src/labs_examples/conditions_loops/labs/Exercise_04.java

```diff
@@ -6,7 +6,19 @@
  *      Using a "for-loop", print out all even numbers from 1-100.
  *
  */
-
+import java.util.Scanner;
 public class Exercise_04 {
 
+    public static void main(String[] args) {
+//        Scanner evenNum = new Scanner(System.in);
+//        System.out.println("Enter a number from 1-100");
+//        int evenNumbers = evenNum.nextInt();
```

Comment:
It is advised to always delete unnecessery or unrelarted comments from the code. Also delete any unecessery code itself. 
Hoever, it is a study project, so you could keep some helpfull constructs of alternative implementations.

Reply:
great tip I was trying to play around and do something different but I will also go and clean it up.